index.html

```diff
@@ -1,185 +1,319 @@
 <!DOCTYPE html>
 <html lang="en">
- <head>
-  <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=G-81YD48KVFY"></script>
-  <script>
-   window.dataLayer = window.dataLayer || [];
-   function gtag(){dataLayer.push(arguments);}
-   gtag('js', new Date());
+  <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-81YD48KVFY"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
 
-   gtag('config', 'G-81YD48KVFY');
-  </script>
-  <meta charset="utf-8">
-  <title>Matchoudi's Portfolio homepage</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <script src="https://kit.fontawesome.com/84967187a9.js" crossorigin="anonymous"></script>
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-  <link rel="stylesheet" href="css/style.css">
-  <link rel="stylesheet" href="css/desktop_style.css">
- </head>
- <body>
-  <header>
-   <a class="icon" href="">My Logo</a>
-   <nav>
-    <ul>
-     <li><a href="#portfolio">Portfolio</a></li>
-     <li><a href="#about">About</a></li>
-     <li><a href="#contact">Contact</a></li>
-    </ul> 
-    <div class="tonav">
-     <i class="fa fa-bars"></i>
-    </div>
-   </nav>
```

Comment:
- [ ] The navbar doesn't show up on the desktop version. Kindly fix it.
<img width="880" alt="image" src="https://user-images.githubusercontent.com/77190203/152020764-3cc81302-8b13-43a8-8406-0191c8fe38f4.png">